Ex1/main.java

```diff
@@ -31,7 +31,9 @@ public static void main(String[] args) {
                     addContact(phone_book);
                     break;
                 case "2":
-                    delContact(phone_book);
+                   System.out.println("enter name of contact you want to remove:");
```

Comment:
Let's keep this switch case clean,
Get name inside function, the delContact function signature should only contain phone_book